src/app/teams/teams-view.component.html

```diff
@@ -10,6 +10,7 @@ <h3 class="margin-lr-3 ellipsis-title">{{team?.name}}</h3>
       <button mat-stroked-button *ngIf="userStatus === 'member'" mat-button class="margin-lr-3" (click)="openInviteMemberDialog()" i18n [disabled]="disableAddingMembers">
         Add Members
       </button>
+      <button *planetAuthorizedRoles mat-raised-button color="accent" class="margin-lr-3" (click)="archiveClick(team)" i18n>Delete</button>
```

Comment:
if button is moved out lets make it last one

Reply:
ok sure thank you.